### PR TITLE
deps: regenerating with new Protobuf (3.7.x)

### PR DESCRIPTION
**Pull request by mpeddada1**
To merge into 3.7.x branch. This PR was generated using Protobuf 3.19.6. Source-Link: https://github.com/googleapis/googleapis/commit/c7ca416c2856aad6a3f7092924e56b8cf0cb2534

### CODE REVIEW

**Review by suztomo — APPROVED** [review]

**Review by losalex — COMMENTED** [review]
Seems build is failed...

**Comment by mpeddada1** [comment]
Hm adding `git config --global --add safe.directory '*'` and `git config --global --add safe.directory /tmpfs/src/github/java-logging` don't seem to do the trick.

File: .kokoro/environment_tests.sh [hunk 1]
```diff
@@ -27,6 +27,7 @@ fi
 # make sure submodule is up to date
 cd "$PROJECT_ROOT"
 git submodule update --init --recursive
+git config --global --add safe.directory '*'
```
Comment (daniel-sanche) [inline .kokoro/environment_tests.sh:30]: It looks like this should be moved up a line. It needs to run before `git submodule update`

Comment (mpeddada1) [reply .kokoro/environment_tests.sh:30]: oh! good catch. I'll modify this.

Comment (daniel-sanche) [reply .kokoro/environment_tests.sh:29]: It looks like this fixed that issue, but there's one more change you'll need to pull in from the `main` branch. It looks like the `python3.6` references in this file need to be changed to `python3` (3.6 is too out of date for the test container)

image

Comment (daniel-sanche) [reply .kokoro/environment_tests.sh:29]: I'm not sure about the failing `GraalVM` tests though

Comment (losalex) [reply .kokoro/environment_tests.sh:29]: thanks @daniel-sanche for comment - indeed, this seems as an issue!
Asd for `GraalVM`, I believe for now those can be ignored

Comment (mpeddada1) [reply .kokoro/environment_tests.sh:29]: Applying this change seemed to have worked!

**Comment by mpeddada1** [comment]
Thank you both so much for troubleshooting this! Merging this PR now.

**Pull request merged**